CHANGELOG.adoc

```diff
@@ -89,6 +89,8 @@ Introducing a generator of builders aimed to help creating view programmatically
 - https://github.com/eclipse-sirius/sirius-components/issues/1989[#1979] [diagram] Contribute a new way to render diagrams to evaluate an alternate layouting strategy
 - https://github.com/eclipse-sirius/sirius-components/issues/1996[#1996] [explorer] Add an Expand All menu item contribution to the Explorer View.
 - https://github.com/eclipse-sirius/sirius-components/issues/1966[#1966] [view] Add Selection Description in the View DSL
+- https://github.com/eclipse-sirius/sirius-components/issues/2015[#2015] [form] We add to the payload the possibility to return a list of messages, these messages will be displayed on the UI.
```

Comment:
Maybe a screenshot with the new UI, ideally showing multiple messages.

Reply:
It might be missing an example on how to use this, at least from inside a Java service. It's pretty artificial, but I used it in `DomainAttributeServices` to test the stacking and rendering of the various levels:

https://github.com/eclipse-sirius/sirius-components/assets/10608/e87262b4-a6d1-48cd-b19b-bc3a10a4aedb

Reply:
I added the exemple and changed the changelog